wlauto/workloads/adobereader/uiauto/app/src/main/java/com/arm/wlauto/uiauto/adobereader/UiAutomation.java

```diff
@@ -102,6 +102,10 @@ private void dismissWelcomeView() throws Exception {
         if (!onboarding_finish_button.exists()) {
             welcomeView.swipeLeft(10);
         }
+        
+        if (!onboarding_finish_button.exists()) {
+        return;
+        }
```

Comment:
That's just to make sure were on the correct screen for the rest of the workload is it not? 

Nothing in this method is key, if the welcome view is not visible. 

The OpenFile function will just fire up.  

You're the boss though.